src/components/NavBar.js

```diff
@@ -27,26 +31,110 @@ const Icon = styled.i`
   font-size: 2rem;
 `;
 
-const NavBar = () => {
-  return (
-    <NavBarContainer>
-      <StyledLink to="/LikedCareers">
-        <NavBarItem>
-          <Icon className="material-icons">playlist_add_check</Icon>
-        </NavBarItem>
-      </StyledLink>
-      <StyledLink to="/Careers">
-        <NavBarItem>
-          <Icon className="material-icons">view_carousel</Icon>
-        </NavBarItem>
-      </StyledLink>
-      <StyledLink to="/Settings">
-        <NavBarItem>
-          <Icon className="material-icons">settings</Icon>
-        </NavBarItem>
-      </StyledLink>
-    </NavBarContainer>
-  );
-};
+const SettingsContainer = styled.div`
+  color: black;
+  background-color: white;
+  width: 100%;
+  transform: ${props =>
+    props.settings ? 'translateY(0)' : 'translateY(-100%)'};
+  transition: all cubic-bezier(0.47, 0, 0.75, 0.72) 0.3s;
+  position: absolute;
+  top: 4rem;
+  right: 0;
+  box-shadow: 0 0.7rem 1.2rem rgba(0, 0, 0, 0.09),
+    0 0.4rem 0.4rem rgba(0, 0, 0, 0.16);
+`;
+
+const Wrapper = styled.div`
+  position: relative;
+`;
+
+const SettingsList = styled.ul`
+  &:last-child {
+    border-bottom: none;
+  }
+`;
+
+const SettlingsListItem = styled.li`
+  height: 4rem;
+  border-bottom: 0.1rem solid gainsboro;
+  border-width: thin;
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  font-size: 1.2rem;
+  color: #455a64;
+  &:active {
+    background-color: gainsboro;
+  }
+`;
+
+const ListItemText = styled.p`
+  text-align: right;
+  padding-right: 5%;
+`;
+
+const ListItemIcon = styled.p`
+  padding-right: 10%;
```

Comment:
As discussed in person, instead of 10% padding right I think both the navbar and the settings items should have a set margin in rems or percent so they align nicely :triangular_ruler: .